test/org/javarosa/core/model/utils/test/DateUtilsTests.java

```diff
@@ -331,6 +334,40 @@ private void testCycle(Date in) {
 		}
 
 	}
+
+	public void testFormatting() {
```

Comment:
I did convert all tabs to spaces in this file (perhaps before I thought of the history obscuration I mentioned above).